### PR TITLE
VCH install timeout set to 30m for 5-21-Datastore-Path

### PR DESCRIPTION
**Pull request by cgtexmex**
Increases timeout from default 3m to 30m

Toward #7718 

[skip ci]

### CODE REVIEW

File: tests/manual-test-cases/Group5-Functional-Tests/5-21-Datastore-Path.robot [hunk 1]
```diff
@@ -43,7 +43,7 @@ Setup Suite ESX
     Remove Environment Variable  BRIDGE_NETWORK
     Remove Environment Variable  PUBLIC_NETWORK
 
-    ${output}=  Install VIC Appliance To Test Server  certs=${false}
+    ${output}=  Install VIC Appliance To Test Server  certs=${false}  additional-args=--timeout=%{TEST_TIMEOUT}
```
Comment (anchal-agrawal) [inline tests/manual-test-cases/Group5-Functional-Tests/5-21-Datastore-Path.robot]: This keyword runs `vic-machine create` with `--timeout=%{TEST_TIMEOUT}`: https://github.com/vmware/vic/blob/master/tests/resources/VCH-Util.robot#L340

We're setting `TEST_TIMEOUT` to 30m on line 39, so if I'm understanding correctly, I don't think we need to change this line.

File: tests/manual-test-cases/Group5-Functional-Tests/5-21-Datastore-Path.robot [hunk 2]
```diff
@@ -57,7 +57,7 @@ Datastore - DS Scheme Specified in Volume Store
 
     Log To Console  \nRunning custom vic-machine create - with DS Scheme in Volume Store
     # Need to run custom vic-machine create to specify volume store with DS scheme
-    ${output}=  Run  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME} --target=%{TEST_URL_ARRAY} --user=%{TEST_USERNAME} --bridge-network=%{BRIDGE_NETWORK} --public-network=%{PUBLIC_NETWORK} --image-store=%{TEST_DATASTORE} --volume-store=${dsScheme}%{TEST_DATASTORE}/images:default --password=%{TEST_PASSWORD} --appliance-iso=bin/appliance.iso --bootstrap-iso=bin/bootstrap.iso --insecure-registry harbor.ci.drone.local --force --kv
+    ${output}=  Run  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME} --target=%{TEST_URL_ARRAY} --user=%{TEST_USERNAME} --bridge-network=%{BRIDGE_NETWORK} --public-network=%{PUBLIC_NETWORK} --image-store=%{TEST_DATASTORE} --volume-store=${dsScheme}%{TEST_DATASTORE}/images:default --password=%{TEST_PASSWORD} --appliance-iso=bin/appliance.iso --bootstrap-iso=bin/bootstrap.iso --insecure-registry harbor.ci.drone.local --force --kv --timeout=%{TEST_TIMEOUT}
```
Comment (anchal-agrawal) [inline tests/manual-test-cases/Group5-Functional-Tests/5-21-Datastore-Path.robot]: 👍

File: tests/manual-test-cases/Group5-Functional-Tests/5-21-Datastore-Path.robot (hunk 1 above)
Comment (cgtexmex) [reply tests/manual-test-cases/Group5-Functional-Tests/5-21-Datastore-Path.robot]: Yep agreed -- I got lost in the `VCH-Util.robot` redirection...and also confused the 2nd & 3rd tests...

**Comment by cgtexmex** [comment]
@anchal-agrawal fixed...

**Pull request merged**